apps/vr-tests-web-components/src/stories/accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { default as parse } from 'html-react-parser';
+import { Steps, StoryWright } from 'storywright';
+import { accordionDefinition, accordionItemDefinition, FluentDesignSystem } from '@fluentui/web-components';
+import { DARK_MODE, getStoryVariant, RTL } from '../../utilities/WCThemeDecorator.js';
+
+accordionDefinition.define(FluentDesignSystem.registry);
+accordionItemDefinition.define(FluentDesignSystem.registry);
+
+export default {
+  title: 'Accordion',
+  decorators: [
+    (story: () => React.ReactElement) => {
+      return (
+        <StoryWright steps={new Steps().snapshot('normal', { cropTo: '.testWrapper' }).end()}>
```

Comment:
Are we using React to render the WC VR tests because we need to use `StoryWright`?

Was a little surprised to see React here :)

Reply:
AFAIR storywright supports only REACT as their api

Reply:
Yeah, I think most of this was simply leveraged by Miro to ensure we could quickly get up and running with existing pipelines, etc. Given it's just a matter of rendering to capture, not too worried here - any delta is going to require more deviation in the build and I don't want to go down that road and add to @Hotell and @TristanWatanabe's workload more than we have :)